implementations/rust/ockam/ockam_vault/src/symmetric_impl.rs

```diff
@@ -109,7 +109,8 @@ mod tests {
         assert!(res.is_ok());
         let plaintext = res.unwrap();
         assert_eq!(plaintext, message.to_vec());
-        ciphertext[0] ^= ciphertext[1];
```

Comment:
The spirit of this statement is to corrupt the ciphertext and show that decryption fails. In CI, sometimes we hit the case where one or both are zero, the ciphertext will remain unchanged, decryption will succeed, and the assertion fails.

This change corrupts the ciphertext in two places with constants, which will always fail

Reply:
@SanjoDeundiak you might find this interesting